packages/now-next/src/index.ts

```diff
@@ -376,8 +376,18 @@ export const build = async ({
     await writeNpmRc(entryPath, process.env.NPM_AUTH_TOKEN);
   }
 
-  console.log('Installing dependencies...');
-  await runNpmInstall(entryPath, [], spawnOpts, meta);
+  const installTime = Date.now();
```

Comment:
This variable should be in the `else` block right?

Reply:
Ya, updated.